wire/internal/wire/wire_test.go

```diff
@@ -59,116 +61,108 @@ func TestWire(t *testing.T) {
 	}
 	wd := filepath.Join(magicGOPATH(), "src")
 
-	t.Run("Generate", func(t *testing.T) {
-		if _, err := os.Stat(filepath.Join(build.Default.GOROOT, "bin", "go")); err != nil {
-			t.Skip("go toolchain not available:", err)
-		}
-		for _, test := range tests {
-			test := test
-			t.Run(test.name, func(t *testing.T) {
-				t.Parallel()
```

Comment:
Why not Parallel() anymore?

Reply:
It complicates the output and doesn't provide much of a speedup, at least on my system. I would guess it's because the tests hit the disk a lot.

Reply:
OK.